src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -62,6 +62,23 @@
  * <pre>
  * &lt;module name=&quot;WhitespaceAfter&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class MyTest {
```

Comment:
Please remove class definition, the less  unnecessary code in example the better.
Your code is not compilable, so extra code does no good.

Reply:
Excellent point. Fixed.